lib/howl/keymap.moon

```diff
@@ -16,7 +16,9 @@
     tab:              'editor-smart-tab'
     shift_tab:        'editor-smart-back-tab'
     backspace:        'editor-delete-back'
+    ctrl_backspace:   'editor-delete-back-word'
```

Comment:
Could you please also add the `osx` equivalents in the section below? I think `meta_backspace` and `meta_delete` should work.